File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q19.py
def main():
    #Input
    radius = float(input("Enter the value of the sphere radius: "))

    #Processing
    volume_sphere = (4 * 3.14 * radius ** 3)/3
    rounded_volume_sphere = round(volume_sphere, 2) 

    #Output
    print(f"The volume of the sphere of radius equal to {volume_sphere} is equal to: {rounded_volume_sphere} v.u")

main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q19.js
import { question, questionFloat } from "readline-sync";

function main(){
    //Input
    const radius = parseFloat(question("Enter the value of the radius: "))

    //Processing
    const volume_sphere = ((4 * 3.14 * radius ** 3) / 3).toFixed(2)
    

    //Output
    console.log(`The volume of radius equal to ${radius} is equal to: ${volume_sphere} v.u`)
}
main()